cmd/livepeer/livepeer_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"math/big"
+	"testing"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/livepeer/go-livepeer/eth"
+	lpTypes "github.com/livepeer/go-livepeer/eth/types"
+	"github.com/livepeer/go-livepeer/pm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSetupOrchestrator(t *testing.T) {
```

Comment:
Need to add the `cmd/livepeer` package to `test.sh` in order to run this test in CI.

Reply:
fixed in b2909ee